prisma/schema.prisma

```diff
@@ -35,13 +37,28 @@ model Shift {
   startTime        DateTime
   endTime          DateTime
   workerSlots      Int
+  blockedWorkers   BlockedWorker[]
 }
 
 model ShiftAssignment {
   shiftUuid  String
   workerUuid String
   shift      Shift  @relation(fields: [shiftUuid], references: [uuid])
   worker     Worker @relation(fields: [workerUuid], references: [uuid])
+  rating     Int?
```

Comment:
- Consider adding @constraint or check validation for rating (1-5 range).
- Consider adding ratedAt DateTime field to track when ratings were submitted